ovn-tester/ovn_utils.py

```diff
@@ -431,28 +431,57 @@ def set_inactivity_probe(self, value):
             ("inactivity_probe", value),
         ).execute()
 
-    def lr_add(self, name):
+    def lr_add(
+        self, name, enabled: bool = False, ext_ids: Optional[Dict] = None
```

Comment:
`enabled: bool = False`

This changes the behavior of the method and is the cause of the CI functional test failure.

Should this either be `enabled: Optional[bool] = None` or `enabled: bool = True`?

Reply:
Thanks for the pointer, you are right. Without passing the `enabled` field, the router has "empty" value in the column which I mistook for the same thing as `enabled=false`

Reply:
So I found a [source on this behavior](https://man7.org/linux/man-pages/man5/ovn-nb.5.html#Logical_Router_TABLE). empty/unset value in `enabled` column is equal to `true`.
Even though the Neutron sets the value to `true` explicitly, I'd say we can drop this change and let it be the default "unset".

Reply:
I've dropped the `enabled` parameter entirely, so I'll mark this conversation as resolved.